src/sentry/static/sentry/app/components/dropdownLink.jsx

```diff
@@ -1,100 +1,220 @@
-import jQuery from 'jquery';
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 
-require('bootstrap/js/dropdown');
-
-const DropdownLink = React.createClass({
-  propTypes: {
+class DropdownLink extends React.Component {
```

Comment:
are the changes to this file copy paste from dropdownReact? or do they need to reviewed on their own merit?

Reply:
It's copy paste apart from a tiny change made in [a separate commit](https://github.com/getsentry/sentry/pull/6592/commits/5c4243df5442e6904e2b898224e1731d52eb96fe)